sounds/migrations/0008_download_license.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-07-12 12:53
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sounds', '0007_auto_20161018_1809'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='download',
+            name='license',
+            field=models.ForeignKey(blank=True, default=None, null=True, on_delete=django.db.models.deletion.CASCADE, to='sounds.License'),
```

Comment:
Ideally the license column should be `NOT NULL`, but while we have packs and sounds in the same table this may not be possible. This will have to be done as part of #914